cardano-prelude-test/cardano-prelude-test.cabal

```diff
@@ -129,10 +93,8 @@ test-suite cardano-prelude-test
                      , template-haskell
                      , text
                      , time
```

Comment:
Are all these deps needed (also for `library`)? You can try compiling with GHC 8.10 + `-Wwarn-unused-packages` (first clean) and see  which are not needed.